apps/neoscan/lib/neoscan/counters/counters.ex

```diff
@@ -4,11 +4,17 @@ defmodule Neoscan.Counters do
   import Ecto.Query, warn: false
   alias Neoscan.Repo
   alias Neoscan.Counter
+  alias Neoscan.AddressBalance
+  alias Neoscan.AddressTransactionBalance
 
   def count_addresses do
     Repo.one(from(c in Counter, where: c.name == "addresses", select: c.value))
   end
 
+  def count_addresses(asset_hash) do
```

Comment:
I am afraid this part will not scale well with the production database, we should probably use a trigger to populate a counter.

Reply:
Thanks for your feedback, I will update it.